content/en/docs/started/workstation/minikube-linux.md

```diff
@@ -93,10 +96,14 @@ Follow these steps to install Kubeflow:
   page](https://github.com/kubeflow/kfctl/releases/tag/{{% kf-latest-version %}}).
```

Comment:
```suggestion
  page](https://github.com/kubeflow/kfctl/releases/).
```

I think this page - https://github.com/kubeflow/kfctl/releases/ - gives users options, in case they don't want to download the latest kfctl.